lib/Config.js

```diff
@@ -237,7 +237,7 @@ function locationConstraintAssert(locationConstraints) {
             assert(typeof details.credentials.secretKey === 'string',
                 'bad config: credentials must include secretKey as string');
         }
-        if (process.env.CI === 'true') {
+        if (process.env.CI_CEPH === 'true') {
```

Comment:
Should update this to `'true'`
https://github.com/scality/cloudserver/blob/33a8e1c4799834f8ef1c75fe82f41920218c6597/eve/main.yml#L232

Reply:
Damn, yes.